pkg/virt-controller/services/template.go

```diff
@@ -679,6 +680,9 @@ func newSidecarContainerRenderer(sidecarName string, vmiSpec *v1.VirtualMachineI
 
 	var mounts []k8sv1.VolumeMount
 	mounts = append(mounts, sidecarVolumeMount())
+	if requestedHookSidecar.DownwardAPI == v1.DeviceInfo {
+		mounts = append(mounts, mountPath(downwardapi.NetworkInfoVolumeName, downwardapi.MountPath))
```

Comment:
It is unsafe to use consts like this, because they may be used in the wrong place.
Returning the object itself would have been safer over exposing the consts.

I will not insist on this.